build-system/common/update-design-review-issues.js

```diff
@@ -0,0 +1,555 @@
+  resultDate.setDate(
+    resultDate.getDate() +
+      (weeks - 1) * 7 +
+      ((7 + dayOfWeek - date.getDay()) % 7)
+  );
+  return resultDate;
+}
+
+/**
+ * @param {Date} date
+ * @param {string} startYyyyMmDd
+ * @return {Array<string>}
+ */
+function getRotation(date, startYyyyMmDd) {
+  const [year, month, day] = startYyyyMmDd
+    .split('-')
+    .map((n) => parseInt(n, 10));
+  const start = new Date(year, month - 1, day);
+  const dateBeginningOfDay = new Date(
+    date.getFullYear(),
+    date.getMonth(),
+    date.getDate()
+  );
+  const weeks = Math.round(
+    // @ts-ignore date calc
+    (dateBeginningOfDay - start) / (7 * 24 * 60 * 60 * 1000)
+  );
+  return timeRotationUtc[weeks % timeRotationUtc.length];
+}
+
+const timeZ = (yyyy, mm, dd, hours, minutes) =>
+  `${yyyy + mm + dd}T${leadingZero(hours) + leadingZero(minutes)}Z`;
+
+/**
+ * @return {Object}
+ */
+function getNextIssueData() {
+  const today = new Date();
+
+  // if we run on the same day of week, we need to skip one day to calculate
+  // properly
+  today.setDate(today.getDate() + 1);
+
+  const nextDay = getNextDayOfWeek(today, dayOfWeek, generateWeeksFromNow);
+  const [region, timeUtcNoDst] = getRotation(
+    nextDay,
+    timeRotationStartYyyyMmDd
+  );
+
+  const [hoursUnadjusted, minutes] = timeUtcNoDst.split(':').map(Number);
+  const hours = hoursUnadjusted - (isDaylightSavingsUsa(nextDay) ? 1 : 0);
+
+  const yyyy = nextDay.getFullYear();
+  const mm = leadingZero(nextDay.getMonth() + 1);
+  const dd = leadingZero(nextDay.getDate());
+
+  const timeUtc = `${leadingZero(hours)}:${leadingZero(minutes)}`;
+
+  const timeUrl = `https://www.timeanddate.com/worldclock/meeting.html?year=${yyyy}&month=${mm}&day=${dd}&iv=0`;
+
+  const startZ = timeZ(yyyy, mm, dd, hours, minutes);
+  const endZ = timeZ(yyyy, mm, dd, hours + sessionDurationHours, minutes);
+
+  const calendarUrl = `http://www.google.com/calendar/event?action=TEMPLATE&text=${encodeURIComponent(
+    calendarEventTitle
+  )}&dates=${startZ}/${endZ}&details=${encodeURIComponent(
+    calendarEventDetails
+  )}`;
+
+  const templateData = {
+    yyyyMmDd: `${yyyy}-${mm}-${dd}`,
+    timeUtc,
+    region,
+    timeUrl,
+    calendarUrl,
+  };
+
+  const title = createTitle(templateData);
+  const body = createBody(templateData);
+
+  return {title, labels, body};
+}
+
+/**
+ * @param {string} key
+ * @return {string}
+ */
+function env(key) {
+  if (!(key in process.env)) {
+    throw new Error(`Missing env variable: ${key}`);
+  }
+  return /** @type {string} */ (process.env[key]);
+}
+
+const datetimeFromTitleRegexp = /(\d{4}-\d{2}-\d{2}) (\d{1,2}[:]\d{2})/;
+
+/**
+ * @param {string} title
+ * @return {number}
+ */
+function getSessionDateFromTitle(title) {
+  const match = title.match(datetimeFromTitleRegexp);
+  if (!match) {
+    throw new Error(`Could not get date from title: ${title}`);
+  }
+  const [
+    // eslint-disable-next-line no-unused-vars
+    unusedFullMatch,
```

Comment:
To handle both TypeScript and eslint:

```suggestion
    // @ts-ignore
    unusedFullMatch, // eslint-disable-line no-unused-vars
```